CONTRIBUTING.md

```diff
@@ -86,9 +86,6 @@ https://github.com/discourse/discourse/blob/master/docs/code-of-conduct.md ?
 
 ## ThoughtWork's role
 
-ThoughtWorks seeds the community that builds CoyIM. We seed the development, investing our own resources: We provide a team of software delivery experts to lay the foundation for the project. We use our network and contacts to approach customers and users.
-
-ThoughtWorks started building CoyIM because it is right. In combining our passion for defending a free internet and our capability to deliver software, we build software to counter widespread mass surveillance of email communication.
-
-It is not ThoughWork's goal to make money from CoyIM. The reasons are multiple, but at the end of the day, we believe that our goals of ensuring digital privacy and anonymity for every person on the planet can best be achieved if CoyIM puts end users before a revenue stream.
+ThoughtWorks started the development of CoyIM on October 2015. The company invested its own resources and provided a team of software delivery experts to lay the foundation for the project.
 
+The goal wasn't to make money from CoyIM, but to combine our capability to deliver software and our passion for defending a free Internet. We want to build software to counter widespread mass surveillance and ensure digital privacy and anonymity for every person on the planet.
```

Comment:
Hey!

What about:
"ThoughtWork's goal wasn't to make money from CoyIM, but to combine their capability to deliver software with our passion for defending a free Internet. Since then, we wanted to build software to counter the widespread of mass surveillance, and ensure digital privacy and anonymity for every person on the planet."

what do you think?

Reply:
Thank you for the suggestion, @claucece.
Now, if we want to tell a story, I think we need to explain TW -> CAD transition.

Reply:
We should probably try to avoid talking too much publicly about the TW -> CAD transition. =)

Reply:
I understand. The description is fine, then.

If the i18n files are not going to get sync errors, I think this PR is good to be merged. :+1: